tests/foreman/cli/test_classparameters.py

```diff
@@ -166,11 +198,14 @@ def test_positive_list_parameters_by_host_name(self):
             'id': sc_param_id,
         })
         self.assertEqual(sc_param['override'], True)
+        organization = entities.Organization(id=self.org['id']).search()[0]
```

Comment:
Why do you use searching here? I bet you meant 
```python
organization = entities.Organization(id=self.org['id']).read()
```
But that shouldn't be the issue - you don't even need to read an organization, according to tests the only usage of organization is during Host initialization. nailgun works with entity fields in 2 ways - you can either set them to entity or with entity id. As you already have entity id and there's no other usages of entity - id should be enough:
```python
host = entities.Host(organization=self.org['id']).create()
```